src/client/pages/OriginalDataPoint/components/DataSources/DataSources.tsx

```diff
@@ -39,101 +34,26 @@ const DataSources: React.FC<Props> = (props) => {
         <div className="fra-table__scroll-wrapper odp__data-source-table-wrapper">
           <table className="fra-table">
             <tbody>
-              <tr>
-                {print && (
-                  <th className="fra-table__header-cell odp__year-column" rowSpan={3}>
-                    {originalDataPoint.year}
-                  </th>
-                )}
-                <th className="fra-table__header-cell-left">{t('nationalDataPoint.references')}</th>
-                <td className="fra-table__cell-left odp__data-source-input-column">
-                  {isDisabled && (
-                    <div className="vgtf__textarea">
-                      <MarkdownPreview value={originalDataPoint.dataSourceReferences ?? ''} />
-                    </div>
-                  )}
-
-                  {!isDisabled && (
-                    <EditorWYSIWYG
-                      onChange={(value) => {
-                        const dataSourceReferences = Objects.isEmpty(value) ? null : value
-                        const originalDataPointUpdate = { ...originalDataPoint, dataSourceReferences }
-                        updateOriginalDataPoint(originalDataPointUpdate)
-                      }}
-                      options={editorOptions}
-                      value={originalDataPoint.dataSourceReferences ?? ''}
-                    />
-                  )}
-                </td>
-                {displayReviewIndicator && (
-                  <td className="fra-table__review-cell no-print">
-                    <ReviewIndicator
-                      title={t('nationalDataPoint.references')}
-                      subtitle={t('nationalDataPoint.dataSources')}
-                      topicKey={Topics.getOdpReviewTopicKey(originalDataPoint.id, 'dataSourceReferences')}
-                    />
-                  </td>
-                )}
-              </tr>
+              <References
+                originalDataPoint={originalDataPoint}
+                updateOriginalDataPoint={updateOriginalDataPoint}
+                reviewIndicator={reviewIndicator}
+                disabled={disabled}
+              />
```

Comment:
The code is only moved to components

Should we use DataGrid instead of table?